c/properties/coverage-error-call.prp

```diff
@@ -0,0 +1,2 @@
+COVER( init(main()), FQL(COVER EDGES(@FUNC(__VERIFIER_error))) )
```

Comment:
To be on the safe site, I just checked the FQL semantics again,
and I think it should be `@CALL(__VERIFIER_error)`.

`@FUNC(..)` describes all statements **of** a function, and
`@CALL(..)` describes all call sites of a function.